css/searchbar.css

```diff
@@ -11,6 +11,7 @@
 	-webkit-user-select: none;
 	max-height: calc(100% - 125px);
 	overflow-y: auto;
+	padding-bottom: 0.5em;
```

Comment:
This should probably only be applied when the searchbar contains > 0 results, otherwise it looks strange:

<img width="799" alt="Screen Shot 2020-07-16 at 11 09 39 AM" src="https://user-images.githubusercontent.com/10314059/87695251-df8d9680-c754-11ea-86b8-ceeb391e2e34.png">

I can't think of an easy way to do this at the moment.

Reply:
Emm, maybe `.item:last-child { margin-bottom: 0.5em; }` ?

Reply:
The items are nested inside a set of containers depending on their type:

```
<div id="searchbar">
    <div class="searchbar-plugin-container">
        <div class="searchbar-item">
    <div class="searchbar-plugin-container">
        <div class="searchbar-item">
```

So it would need to be something like `#searchbar .searchbar-plugin-container:not(:empty):last-of-type .item:last-child`, but I think :last-of-type only matches based on the tag name.

Reply:
Removed